### ISSUE TITLE
How to verify the data of a car at auction to be able to buy it?

### ISSUE THREAD

**Issue by cotetowemix** [cotetowemix/Cars#1]
I am very happy because I have finally been able to raise a fairly considerable amount of money to buy my first car, clearly it is not enough for a new car so I will have to buy a used car, I was evaluating several places to do it and many people have recommended me buy a car at auction, there I can get a very good used car at an incredible price, my fear is, how can I know if the car I am going to buy at auction does not have any legal problems and the papers are real? Because it's going from person to person, I'm afraid to buy a car that has false or illegal papers. Does anyone know how to check the information on the auctioned car?

**Comment by Anastasiya900** [comment]
I have never bought a car at an auction, but many of my friends buy their cars this way all the time. As far as I know, they never had any problems. In addition, some auctions provide the service of checking a car by VIN code, so the risks are minimal. So do not be afraid, check the car and be calm.

**Comment by Franciscos33** [comment]
Buying a used car at an auction can be a good way to get a good deal, but it's important to be vigilant and check the car's information to avoid possible problems. Many companies provide a vehicle history check service based on its VIN number. This report contains information about previous owners, repair and service history, accidents and other significant events that could affect the condition of the car.

**Comment by Edmond232** [comment]
It's always worth exercising caution when buying any used car from an auction or private seller, either in person or online. So having access to reliable data about its past will help you make informed decisions based on facts, not just assumptions. Knowing everything ahead of time, before you make your purchase, can save you money on costly repairs if something was missed during the pre-sale inspection. By checking the VIN through [https://carcheckvin.com/](https://carcheckvin.com/), buyers can be assured that they have complete peace of mind when buying their next used car.

**Comment by FrankGinalbe** [comment]
Absolutely, you are correct. When buying a used car at an auction or from any other source, it is crucial to be vigilant and gather as much information as possible about the vehicle's history to make an informed decision. Vehicle history reports based on the VIN number can provide valuable insights into the car's background.

**Comment by Edmond232** [comment]
It always pays to exercise caution when buying any used car at an auction or from a private seller, in person or online. Thus, accessing reliable data about his past will help you make informed decisions based on facts, not just speculation. Knowing everything in advance, before making a purchase, you can save money on expensive repairs if something was missed during the pre-sale inspection.
